.env.local.sample

```diff
@@ -37,6 +37,6 @@ MAIL_FROM_ADDRESS=staging@terramatch.org
 # recipients are encoded in the X-Original-Recipients header
 MAIL_RECIPIENTS=
 
-TRANSIFEX_TOKEN=
+TRANSIFEX_TOKEN=1/dada8aab9ed6bba68805fcab6349f2e69d3b1367
```

Comment:
Note: this will be required for local developers to have configured now that we're using the translation service in report title generation.